docs/docs/misc/migration_notes.md

```diff
@@ -6,8 +6,31 @@ keywords: [sandbox, cli, aztec, notes, migration, updating, upgrading]
 
 Aztec is in full-speed development. Literally every version breaks compatibility with the previous ones. This page attempts to target errors and difficulties you might encounter when upgrading, and how to resolve them.
 
+## TBD
+
+### [Aztec.nr] Public storage historical read API improvement
+
+`history::public_value_inclusion::prove_public_value_inclusion` has been renamed to `history::storage::storage_historical_read`, and it's API changed slightly. Instead of receiving a `value` parameter it now returns the historical value stored at that slot.
```

Comment:
```suggestion
`history::public_value_inclusion::prove_public_value_inclusion` has been renamed to `history::storage::storage_historical_read`, and its API changed slightly. Instead of receiving a `value` parameter it now returns the historical value stored at that slot.
```

Reply:
See my comment from earlier. Think it is not clear from the name that it is public storage.